src/@types/alltypes.d.ts

```diff
@@ -2,6 +2,7 @@ declare module '@variate/engine';
 
 type ProviderStateType = {
   variate: any
+  audience: object
```

Comment:
Careful, we renamed audiences to segments recently

Reply:
What, when did that happen?

I should change this name.

Reply:
I'll change this naming